src/main/resources/org/mapstruct/intellij/messages/MapStructBundle.properties

```diff
@@ -15,13 +15,15 @@ inspection.no.source.property=No source property defined
 inspection.more.than.one.source.property=More than one source property defined
 inspection.more.than.one.default.source.property=More than one default source property defined
 inspection.not.null.checkable.property.source.used.with.default.property={0} property used with {1}
+inspection.java.expression.unnecessary.whitespace=Remove unnecessary whitespaces {0} {1}
```

Comment:
To be consistent, this should not have an intent in its name but only a description of the current state: `Unnecessary whitespaces {0} {1}`

Reply:
I agree. I changed the implementation to use the properties instead of an string.